### DISCUSSION TITLE
Видно ли использование zapret провайдеру?

### DISCUSSION THREAD

**Discussion by SpiritOTHawk** [bol-van/zapret#958] [General]
`
Роскомнадзор (РКН) хочет обязать операторов связи предоставлять информацию, которая позволит идентифицировать средства связи и оборудование абонентов в интернете. За счёт этого ведомство сможет получить сведения о пользователях, которые заходят на сайты, заблокированные в России. Об этом сообщили «Ведомости» со ссылкой на проект приказа РКН.
`
И можно ли будет скрыть факт использования zapret в случае, если введут наказание за обход блокировок?

**Comment by bol-van** [comment]
Пока что от этого сильно несет желтизной
Никакой технической конкретики, только размытые фразы и чьи-то догадки

**Comment by SpiritOTHawk** [comment]
`
"В целях эффективного выявления интернет‑трафика, обеспечивающего доступ к противоправным интернет‑ресурсам, для последующего ограничения доступа к ним техническими средствами противодействия угрозам ТСПУ проводится анализ интернет‑трафика. Принадлежность интернет‑трафика пользовательскому оборудованию (оконечному оборудованию) конкретного пользователя определяется по сетевому адресу. Проект приказа устанавливает обязанность операторов связи представлять в Роскомнадзор информацию, позволяющую идентифицировать средства связи и пользовательское оборудование (оконечное оборудование) в сети «Интернет», и выделенных для их использования сетевых адресах, а также сведения об изменении указанной информации."
`

**Reply by bol-van** [reply]
Можно только догадываться что они там собираются делать. Но по логике они могут захотеть адаптивно выявлять VPNы и банить их динамически. Так сделано в китае, в туркменистане и других.
Это следующий шаг в блокировках.
